docker/php/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6.22-fpm-alpine
+RUN apk upgrade --update && apk add \
+  coreutils \
+  freetype-dev \
+  libjpeg-turbo-dev \
+  libltdl \
+  libmcrypt-dev \
+  libpng-dev
```

Comment:
That's right! I've updated the installed dependencies. Had to build unrtf as it's not available as an apk for alpine linux.